src/ex5/server/db/migrations/20220619174307-create-items.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Items', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      ItemName: {
```

Comment:
consider: btw it's better to have concise column name like name - because it's obvious that this name is related to item - because it's under the items table!